.eslintrc.js

```diff
@@ -50,22 +50,25 @@ module.exports = {
         ' ',
       ],
     ],
-    'import/no-unresolved': ERROR,
-    'import/no-extraneous-dependencies': [ERROR, {"includeTypes": true}],
-    "react/require-default-props": [WARNING, {
-      "functions": "defaultArguments"
-    }],
-    'react/jsx-filename-extension': [WARNING, { extensions: ['.js', '.jsx', '.ts', '.tsx'] }]
+    'import/no-unresolved': [ERROR, {ignore: ['^@docusaurus']}],
```

Comment:
these changes should have a rationale

Reply:
It is my mistake of reformatting change and actual change in one single commmit.
The only actual change is ignore @docusaurus because somehow ESLint cannot properly resolve it. Everything else is just reformatting.